shared/src/commonMain/kotlin/com/example/tasky/agenda/domain/AgendaRepository.kt

```diff
@@ -38,6 +49,12 @@ interface IAgendaRepository {
         photos: List<ByteArray>,
     ): ResultWrapper<Event, BaseError>
 
+    suspend fun createLocalEvent(): Event
+
+    suspend fun createLocalReminder(): Reminder
+
+    suspend fun createLocalTask(): Task
```

Comment:
What is your thinking here? 🤔 

Let me give you my perspective: I see when a user taps the “+” button to add a new agenda item as a valid reason to create a new model and save it to the remote server. Thus, I don’t see the purpose of having these functions here to create local models. Will you have another three to call the remote endpoints?

I might not be seeing something that you are 😄

Reply:
It seems that I misunderstand the requirement. I think that the agenda would only be created in remote server when the user presses the save button, so I only create it only in local at first, which can avoid unnecessary api calls to remote server.